cyfi/data/satellite_data.py

```diff
@@ -372,10 +374,18 @@ def download_row(
         imagery_dir (Path): Image cache directory for a specific satellite
             source and bounding box size
         config (FeaturesConfig): Features config
+        log_level (Optional[int]): Log level to use for this download.
+            If None, makes no changes to the logger. Defaults to None.
 
     Returns:
         bool: True if the download was successful, False otherwise
     """
+    # process_map runs in separate worker processes
+    # update to use the same log level specified in the CLI
+    if log_level is not None:
```

Comment:
Since `process_map` runs in a separate worker process, each has its own logger configuration. These weren't inheriting our log level input to the CLI, so were always showing debug logs. Now they'll use the log level we set in the CLI.